Dockerfiles/Dockerfile_Ubuntu

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:23.10 
+
+RUN apt-get update
+RUN apt-get install -y build-essential
+RUN apt-get install -y netcdf-bin libnetcdf-c++4-dev libboost-all-dev libeigen3-dev cmake
+RUN apt-get install -y python3-netcdf4
+
+WORKDIR /home/nextsim
+COPY . /home/nextsim
+
+RUN rm -r build
```

Comment:
you might need to include `-f` flag here because on a clean clone of the github repo build would not already exist.

```suggestion
RUN rm -rf build
```